tests/test_venue_request_v2.py

```diff
@@ -485,7 +485,7 @@ def test_venue_recruitment(self, client, test_client, selenium, request_page, ve
         # assert reply_row
         # buttons = reply_row.find_elements_by_class_name('btn-xs')
         # assert [btn for btn in buttons if btn.text == 'Recruitment']
-        reviewer_details = '''reviewer_candidate1_v2@mail.com, Reviewer One\nreviewer_candidate2_v2@mail.com, Reviewer Two'''
+        reviewer_details = '''reviewer_candidate1_v2@mail.com, Reviewer One\nReviewer_Candidate2_v2@mail.com, Reviewer Two'''
```

Comment:
I don't understand why the email was added to the invited group with no error but the invitation link still had the email in capital letter, any ideas?

Reply:
It was because we were not changing the email to lowercase at any point, but for some reason `add_members_to_group()` add the email in lowercase. Not really sure why this is the case.

Reply:
if I do: 
`openreview_client.add_members_to_group('NeurIPS.cc/2023/Conference/Senior_Area_Chairs/Invited', ['Celeste.Martinez@gmail.com'])`, the email `celeste.martinez@gmail.com` is added to the group, but everywhere else in the recruitment function we were still using `Celeste.Martinez@gmail.com`

Reply:
@carlosmondra is the backend converting emails to lowercase when they are added as members of a group? 

this is the code for `add_members_to_group`: https://github.com/openreview/openreview-py/blob/master/openreview/api/client.py#L1470